ls1intum/Artemis
# `Programming exercises`: Improve readability of build output in the online editor

### Checklist
#### General


#### Client

### Motivation and Context
Closes #5868 

### Description
Replaces hard-coded colour values with variable ones.

The changes to `_{default,dark}-variables.scss` are to make the colours between dark and light mode consistent:
- they use very slightly off-white/black for the main text, and
- a very slightly off-black/white for the background.

### Steps for Testing

Prerequisites:
- 1 Instructor
- 1 Programming exercise with the online editor enabled

1. Submit a solution to the programming exercise that fails to compile.
2. Look at the build output below the online editor. It should be readable in both light and dark mode.

#### Code Review
#### Manual Tests

### Screenshots
Before:
![Screenshot 2022-11-23 at 12-06-45 Code Editor](https://user-images.githubusercontent.com/64250573/203531885-b3263d43-7ea3-4371-bf5a-7ea62a10b647.png)

After:
![Screenshot 2022-11-25 at 21-08-49 Code Editor](https://user-images.githubusercontent.com/64250573/204051701-030ed998-9f2c-4105-a874-f2a38a801994.png)
![Screenshot 2022-11-25 at 21-09-06 Code Editor](https://user-images.githubusercontent.com/64250573/204051704-827a7bfb-b0df-4a32-99e9-aa72fbc00f19.png)
